test/Microsoft.AspNetCore.Server.KestrelTests/MemoryPoolIteratorTests.cs

```diff
@@ -606,6 +606,68 @@ public void GetsKnownMethod(string input, bool expectedResult, string expectedKn
         }
 
         [Theory]
+        [InlineData("http://abc", true, MemoryPoolIteratorExtensions.HttpScheme)]
+        [InlineData("http:// ", true, MemoryPoolIteratorExtensions.HttpScheme)]
+        [InlineData("http:/  ", false, null)]
+        [InlineData("HtTp:// ", false, null)]
+        [InlineData("https://", true, MemoryPoolIteratorExtensions.HttpsScheme)]
+        [InlineData("https://abc", true, MemoryPoolIteratorExtensions.HttpsScheme)]
```

Comment:
Test same conditions on http and https. You're not testing https:/ nor followed by space.

For both: `http:` and `https:`